Pr1/stam.py

```diff
@@ -0,0 +1,41 @@
+import sys
+
+
+FILE_NAME = 1
+FILE_NAME_NEW = "D:/Python Training/Pr1/stam_new.txt"
+
+
+def write_to_file(file_name, FILE_NAME_NEW, first_name, last_name):
```

Comment:
You should not use a const as an argument to your function.
 if it's a const why have it as an argument?